### PR TITLE
#763 change img src path

### PR DESCRIPTION
**Pull request by mariajaro**
Path to images and icons should not use the public in paths

../../public/images/logoipsum-287.svg" WRONG
/images/logoipsum-287.svg" CORRECT

../../public/assets/icons/filter.svg" WRONG
/assets/icons/filter.svg" CORRECT

Filter icon has Search icon as alt text:

Search icon
Missing alt text:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -108,4 +109,11 @@ <h2 class="d-md-none fw-bold">Your direct access to Noroff's best candidates</h2
...
-</html>
\ No newline at end of file
+</html>
+=======
+      </section>
+    </main>
+    <footer></footer>
+  </body>
+</html>
+>>>>>>> Stashed changes
```
Comment (Olekrr) [inline index.html:119]: You seem to have some stashed changes leftover in your code that needs to be removed

File: pages/listings/index.html [hunk 2]
```diff
@@ -43,6 +43,7 @@ <h1 class="mt-2 py-1 text-center fw-bold">Job Listings</h1>
                 <div class="input-group">
                   <label for="searchListing"></label>
 
+<<<<<<< Updated upstream
```
Comment (Olekrr) [inline pages/listings/index.html:46]: What is this?

File: pages/listings/index.html [hunk 3]
```diff
@@ -73,10 +74,15 @@ <h1 class="mt-2 py-1 text-center fw-bold">Job Listings</h1>
...
-                    alt="Search icon"
+                    src="/assets/icons/filter.svg"
+                    alt="Filter icon"
                     class="icon-sm"
                   />
+=======
+            <div class="dropdown">
+                <button class="btn" type="button" id="dropdownMenuButton1" data-bs-toggle="dropdown" aria-expanded="false">
+                  <img src="/assets/icons/filter.svg" alt="Search icon" class="icon-sm" />
+>>>>>>> Stashed changes
```
Comment (Olekrr) [inline pages/listings/index.html:85]: More stashed changes

**Review by Olekrr — CHANGES_REQUESTED** [review]
Hello Maria, I have reviewed your PR and there are a couple of things you need to look at. Please refer to my comments

File: pages/listings/index.html (hunk 2 above)
Comment (mariajaro) [reply pages/listings/index.html:46]: removed

File: pages/listings/index.html (hunk 3 above)
Comment (mariajaro) [reply pages/listings/index.html:85]: removed

File: index.html [hunk 4]
```diff
@@ -107,5 +108,9 @@ <h2 class="d-md-none fw-bold">Your direct access to Noroff's best candidates</h2
   </main>
   <footer></footer>
 </body>
-
+</html>
+      </section>
+    </main>
+    <footer></footer>
+  </body>
 </html>
```
Comment (Olekrr) [inline index.html:116]: There are still some issues here that needs to be resolved. There seems to be additional section, main, footer, body and html closing tags

File: index.html [hunk 5]
```diff
@@ -94,6 +94,7 @@ <h2 class="d-none d-md-block fw-bold">Your direct access to industry</h2>
               class="btn customButton btn-theme-primary text-theme-light fw-semibold p-3">Register as a student</a>
           </div>
         </div>
+<<<<<<< Updated upstream
```
Comment (Olekrr) [inline index.html:97]: What is this line?

**Review by Olekrr — CHANGES_REQUESTED** [review]
Hi Maria, mostly this is fixed but it looks like there is a bit left. Refer to my comments

File: index.html (hunk 5 above)
Comment (mariajaro) [reply index.html:97]: removed

**Comment by Olekrr** [comment]
This has in the meantime been completed by a different group, as such this task is redundant and will be closed without a merge.

**Pull request closed without merging**